{"name": "test_five", "status": "failed", "statusDetails": {"message": "AssertionError: assert 10 == 8\n +  where 10 = add(5, 5)", "trace": "self = <Work.test_work.TestB object at 0x10b46e358>\n\n    def test_five(self):\n        logging.info(\"This is test_five!!!\")\n>       assert add(5, 5) == 8\nE       assert 10 == 8\nE        +  where 10 = add(5, 5)\n\ntest_work.py:53: AssertionError"}, "attachments": [{"name": "log", "source": "1b471d17-41f7-4bbc-8b0b-9010dcbd057d-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "da7fe84c-d2c8-44ef-95e5-816cedd5ffe9-attachment.txt", "type": "text/plain"}], "start": 1570674115218, "stop": 1570674115219, "uuid": "190334fd-0e4f-4992-b678-7561df5bf387", "historyId": "29300f720fe0109f83442a8fb4bb3911", "testCaseId": "f779b13c2b9c2dc87d78ea24bb4c7d7a", "fullName": "test_work.TestB#test_five", "labels": [{"name": "suite", "value": "test_work"}, {"name": "subSuite", "value": "TestB"}, {"name": "host", "value": "ParsondeMacBook-Pro.local"}, {"name": "thread", "value": "27998-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_work"}]}